src/otx/algo/detection/heads/custom_anchor_generator.py

```diff
@@ -9,12 +9,13 @@
 
 import numpy as np
 import torch
-from mmdet.registry import TASK_UTILS
+from mmengine.registry import TASK_UTILS
 from torch.nn.modules.utils import _pair
 
 
 # This class and its supporting functions below lightly adapted from the mmdet AnchorGenerator available at:
 # https://github.com/open-mmlab/mmdetection/blob/cfd5d3a985b0249de009b67d04f37263e11cdf3d/mmdet/models/task_modules/prior_generators/anchor_generator.py
+@TASK_UTILS.register_module()
```

Comment:
@wonjuleee I want to highlight it will be used to build another anchor generator for RTMDetInst in an upcoming PR. Branching could be an option for handling this, that said, it's worth noting that numerous components currently depend on the mmengine registry. Removing this dependency is not within the scope of this PR. However, it's something we should address comprehensively at a later stage, although not suitable within this PR.